contracts/staking/PoSValidatorManager.sol

```diff
@@ -149,7 +147,7 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
         bytes32 validationID,
         address delegator,
         uint256 delegationAmount
-    ) internal nonReentrant {
+    ) internal nonReentrant returns (bytes32) {
```

Comment:
Can we rename `delegator` to `delegatorAddress`? I was reading through the function and mistook it for a `Delegator` object.